packages/noports_core/test/sshnp/sshnp_core_mocks.dart

```diff
@@ -60,35 +44,35 @@ class StubbedSshnpCore extends SshnpCore with StubbedAsyncInitializationMixin {
 
 /// Stubbed mixin wrapper
 mixin StubbedAsyncInitializationMixin on AsyncInitialization {
-  late FunctionStub _mockCallInitialization;
-  late FunctionStub _mockInitialize;
-  late FunctionStub _mockCompleteInitialization;
+  late FunctionStub<void> _stubbedCallInitialization;
+  late FunctionStub<void> _stubbedInitialize;
+  late FunctionStub<void> _stubbedCompleteInitialization;
 
```

Comment:
Updated the naming convention here, in some cases we aren't doing true mocks, but instead calling the real function and using the Mock class as a wrapper which does the stubbing for us.